src/styles/StyleUtils.ts

```diff
@@ -1012,7 +1023,7 @@ function getEmojiReactionCounterTextStyle(hasUserReacted: boolean): TextStyle |
  *
  * @param direction - The direction of the rotation (CONST.DIRECTION.LEFT or CONST.DIRECTION.RIGHT).
  */
-function getDirectionStyle(direction: string): ViewStyle | CSSProperties {
+function getDirectionStyle(direction: string): ViewStyle {
```

Comment:
Do we have a type for direction? We have consts for this. I think it makes sense to have a type for such kind of variables.
```suggestion
function getDirectionStyle(direction: string): ViewStyle {
```